src/main/groovy/org/transmartproject/batch/batchartifacts/AbstractSplittingItemReader.groovy

```diff
@@ -28,6 +28,7 @@ abstract class AbstractSplittingItemReader<T> extends ItemStreamSupport implemen
     // to be configured
     protected ItemStreamReader<FieldSet> delegate // should not be registered as stream
     EagerLineListener<T> eagerLineListener
+    EarlyItemFilter<T> earlyItemFilter
```

Comment:
You can also had some note saying that the filter happens before the line listener and that the filter must be stateless.